src/backend-ai-app.ts

```diff
@@ -62,7 +62,10 @@ const loadPage = (page, params: Record<string, unknown> = {}) => (dispatch) => {
     break;
   case 'pipeline-job':
     import('./pipeline/components/pipeline-job-view.js');
-    break; */
+    break;
```

Comment:
Merging main branch with fixing conflicts to this branch may have commented out `agent-summary` page by accident.
adding `*/` at the end of L68 will fix this.

Reply:
I resolved it. thank you :)